code/modules/admin/verbs/adminpm.dm

```diff
@@ -46,6 +46,25 @@
 		else		src << "<font color='red'>Error: Private-Message: Client not found. They may have lost connection, so try using an adminhelp!</font>"
 		return
 
+	if (src.handle_spam_prevention(msg,MUTE_ADMINHELP))
+		return
+
+	var/recieve_pm_type = "Player"
+	if(holder)
+		//mod PMs are maroon
+		//PMs sent from admins and mods display their rank
+		if(holder)
```

Comment:
```dm
if (holder)
	if (holder)
```

...?

Reply:
Lol this code has been on Bay for the longest time. I thought I had fixed it when I first wrote tickets.